src/dart/ast.rs

```diff
@@ -113,6 +130,8 @@ pub enum Item {
         interfaces: Vec<Node<Qualified>>,
         members: Vec<Node<ClassMember>>,
     },
+    /// A mixin describes the difference between a class and its superclass
```

Comment:
"the difference" is still suboptimal, how about "A mixin class combines the members of multiple classes."?